### PR TITLE
[ch64] As a User, I can see About us screen

### PR DESCRIPTION
**Pull request by Raksani**
[https://app.clubhouse.io/cobda/story/64/as-a-user-i-can-see-about-us-screen](url)

## Why?

This **PR** is the entry point for the `About Us` screen. And accessible through the `Footer` component. 

## Changes

- Create `About Us` screen.
- Add `About us` link on the `Footer`.

## Screenshot(s)

**Before**
105196496-478ec080-5b6e-11eb-8bb6-57f55272deca

**After**
About us link appears on Footer. And About Us screens was created.
Screenshot 2564-01-31 at 14 32 39

### CODE REVIEW

**Comment by LilBank** [comment]
Quick reminder, a commit message isn't a sentence 😅, no need for a period(.)

**Comment by LilBank** [comment]
Please update PR name.

File: src/components/Footer/styles.scss [hunk 1]
```diff
@@ -1,11 +1,17 @@
-@import '../../styles/main.scss';
+@import "../../styles/main.scss";
```
Comment (LilBank) [inline src/components/Footer/styles.scss]: Single quotes 🤓

File: src/components/Footer/styles.scss [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
   height: rem(32px);
   background-color: $secondary;
 
   @include flex-center();
   @include respond-above(md) {
     height: rem(72px);
   }
+
+  &__item {
+    font-size: 16px;
```
Comment (LilBank) [inline src/components/Footer/styles.scss]: Please convert this into `rem` unit.

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -1,6 +1,10 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './styles.scss';
 
 export default function Footer() {
-  return <div className="footer-container">Footer</div>;
+  return <div className="footer">
+    Footer
+    <Link to="/about-us" className="footer__item">About Us</Link>
+    </div>;
```
Comment (LilBank) [inline src/components/Footer/index.tsx]: @kimvcp Could you please recheck this indention?

**Review by LilBank — CHANGES_REQUESTED** [review]

File: src/router/index.tsx [hunk 4]
```diff
@@ -15,6 +16,9 @@ export default function Routes() {
         <Route exact path="/">
           <Home />
         </Route>
+        <Route exact path="/about-us">
```
Comment (kimvcp) [inline src/router/index.tsx]: Please remove `exact` property from every path except the main one. (`/`)

Comment (Raksani) [reply src/router/index.tsx]: this file will be deleted after changing to Nextjs.

**Comment by LilBank** [comment]
Screen Shot 2564-01-31 at 14 12 45

Please remove the period.

**Comment by LilBank** [comment]
Please update the screenshots.

File: src/screens/AboutUs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Footer from '../../components/Footer';
+import Navbar from '../../components/Navbar';
+import Sidebar from '../../components/Sidebar';
+import './styles.scss';
+
+export default function AboutUs() {
+  return (
+    <>
```
Comment (LilBank) [inline src/screens/AboutUs/index.tsx]: The **AboutUs** should have its own layout. Please check either **Sign up** or **Home** about the implementation. It should be something like `className = layout-about-us`

File: src/screens/AboutUs/styles.scss [hunk 6]
```diff
@@ -0,0 +1 @@
+@import '../../styles/main.scss';
```
Comment (LilBank) [inline src/screens/AboutUs/styles.scss]: This file is outdated (React), please update(Next).

File: src/components/Footer/styles.scss [hunk 7]
```diff
@@ -0,0 +1,17 @@
+@import '../../styles/main.scss';
+
+.footer {
```
Comment (LilBank) [inline src/components/Footer/styles.scss]: `footer`'s height should be remove when there's a layout for **About us**.

**Review by LilBank — CHANGES_REQUESTED** [review]

File: src/components/Footer/index.tsx [hunk 8]
```diff
@@ -1,7 +1,13 @@
 import React from 'react'
+import Link from 'next/link'
 
-const Footer = () => {
-  return <div className="footer-container">Footer</div>
-}
+const Footer = () => (
+  <div className="footer-container">
```
Comment (LilBank) [inline src/components/Footer/index.tsx]: It's a `footer` so I don't think we should name it `footer-container`.

File: src/components/Footer/index.tsx [hunk 9]
```diff
@@ -1,7 +1,13 @@
...
+    Footer
+    <div className="footer__item">
+      <Link href="/about-us">About Us</Link>
```
Comment (LilBank) [inline src/components/Footer/index.tsx]: Please check `navbar` on how to use `<Link>`

File: src/pages/about-us.tsx [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+            <Navbar />
+        </header>
+        <main>
+        </main>
+        <footer>
+            <Footer />
+        </footer>
+    </div>
+)
+export default AboutUs
```
Comment (LilBank) [inline src/pages/about-us.tsx]:
```suggestion
)

export default AboutUs
```

File: src/pages/about-us.tsx [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+    <div className="layout-about-us">
+        <header>
+            <Navbar />
+        </header>
+        <main>
+        </main>
+        <footer>
+            <Footer />
+        </footer>
+    </div>
```
Comment (LilBank) [inline src/pages/about-us.tsx]: Indention. 🤔

File: src/pages/about-us.tsx [hunk 12]
```diff
@@ -0,0 +1,16 @@
+import Footer from '../components/Footer'
+import Navbar from '../components/Navbar'
+
+const AboutUs = () => (
+    <div className="layout-about-us">
+        <header>
+            <Navbar />
+        </header>
+        <main>
+        </main>
```
Comment (LilBank) [inline src/pages/about-us.tsx]: Would be nice if you and a `<div>` with a text in the middle says "About us".

File: src/styles/layouts/_layout-about-us.scss [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+      @include respond-above('lg') {
+        padding: 5% 10%;
+      }
+    }
+  
+    & > footer {
+      grid-area: footer;
+    }
+  }
+
```
Comment (LilBank) [inline src/styles/layouts/_layout-about-us.scss]: Please add newline.

File: src/styles/layouts/_layout-about-us.scss [hunk 14]
```diff
@@ -0,0 +1,46 @@
+.layout-about-us {
```
Comment (LilBank) [inline src/styles/layouts/_layout-about-us.scss:1]: Indention. 🙇🏻‍♂️

**Review by LilBank — CHANGES_REQUESTED** [review]

File: src/styles/components/_footer.scss [hunk 15]
```diff
@@ -1,6 +1,6 @@
 @import '../abstracts/_abstracts-dir';
 
-.footer-container {
+.footer {
```
Comment (LilBank) [inline src/styles/components/_footer.scss:3]: As you can see, we already have `grid` controlling the size, so no need for `respond-above` in here.

File: src/pages/about-us.tsx [hunk 16]
```diff
@@ -0,0 +1,18 @@
+import Footer from '../components/Footer';
```
Comment (LilBank) [inline src/pages/about-us.tsx]: Please remove semicolon from this file.

File: src/components/Footer/index.tsx [hunk 17]
```diff
@@ -1,7 +1,15 @@
...
 
-const Footer = () => {
-  return <div className="footer-container">Footer</div>
-}
+const Footer = () => (
+  <div className="footer">
+    Footer
+    <div className="footer__item">
+      <Link href="/about-us">
+        <a className="footer__link">About Us</a>
```
Comment (LilBank) [inline src/components/Footer/index.tsx:9]: I don't see a class for `footer__link` in `_footer.scss`?

Comment (LilBank) [reply src/components/Footer/index.tsx:9]: Either remove it or add an empty class there and put a comment(`// TODO: For future implementation`).

**Review by LilBank — CHANGES_REQUESTED** [review]

**Review by LilBank — APPROVED** [review]

**Review by kimvcp — APPROVED** [review]

**Pull request merged**